s3_tools/objects/read.py

```diff
@@ -39,9 +39,6 @@ def read_object_to_bytes(bucket: str, key: str) -> bytes:
     s3 = session.client("s3")
     obj = s3.get_object(Bucket=bucket, Key=key)
 
-    if "Body" not in obj:
-        raise KeyError("Missing key 'Body' on response")
-
```

Comment:
Why did we remove the raise of the error?

We need to remove it from the docstring.

Reply:
Remove after checking AWS docs and moto package. Every time that the get_object returns a response the Body key will always be there, even if is empty. Basically, this code was useless.

I always forget about the docstring 🤦